lib/packages/core/lib/src/blocs/conversation/conversation_event.dart

```diff
@@ -32,18 +32,36 @@ class LMChatFetchConversationsEvent extends LMChatConversationEvent {
   /// Number of conversations to be fetched
   final int pageSize;
 
+  final int? minTimestamp;
+  final int? maxTimestamp;
+  final PaginationDirection direction;
```

Comment:
What class is this `PaginationDirection`? Did we make it or it comes with the package?

Reply:
we made it in pagination utils